lib/diagnostic.js

```diff
@@ -41,7 +57,7 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.remove({motto:'As High as Honor'});
```

Comment:
This will remove the entire document rather than just the key.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```